src/cmd/build.rs

```diff
@@ -268,7 +268,7 @@ fn optimize_wasm(crate_metadata: &CrateMetadata) -> Result<OptimizationResult> {
     let mut dest_wasm_file_content = Vec::new();
     dest_wasm_file.read_to_end(&mut dest_wasm_file_content)?;
 
-    let optimized_wasm = get_optimized_wasm(crate_metadata, &optimized, &dest_wasm_file_content)?;
+    let optimized_wasm = do_optimization(crate_metadata, &optimized, &dest_wasm_file_content, 3)?;
```

Comment:
Would be good to make a type here to represent the valid opt levels, but it doesn't necessarily have to be part of this PR

```
 -O                                           execute default optimization 
                                                passes 
   -O0                                          execute no optimization passes 
   -O1                                          execute -O1 optimization passes
                                                (quick&useful opts, useful for 
                                                iteration builds) 
   -O2                                          execute -O2 optimization passes
                                                (most opts, generally gets most
                                                perf) 
   -O3                                          execute -O3 optimization passes
                                                (spends potentially a lot of 
                                                time optimizing) 
   -O4                                          execute -O4 optimization passes
                                                (also flatten the IR, which can
                                                take a lot more time and memory,
                                                but is useful on more nested / 
                                                complex / less-optimized input)
   -Os                                          execute default optimization 
                                                passes, focusing on code size 
   -Oz                                          execute default optimization 
                                                passes, super-focusing on code 
                                                size 
```

Reply:
Good point, I plan on doing the follow-up of exposing these via the cli anyway, so I'll include it then.